src/main/webapp/app/shared/modal/AddMutationModal.tsx

```diff
@@ -1009,13 +1024,19 @@ interface IAddMutationModalFieldProps {
   value: string;
   placeholder: string;
   onChange: (newValue: string) => void;
+  isLoading?: boolean;
 }
 
-function AddMutationModalField({ label, value: value, placeholder, onChange }: IAddMutationModalFieldProps) {
+function AddMutationModalField({ label, value: value, placeholder, onChange, isLoading }: IAddMutationModalFieldProps) {
   return (
     <Row className="align-items-center mb-3">
       <Col className="px-0 col-3 mr-3">
-        <span>{label}</span>
+        <span className="mr-2">{label}</span>
+        {isLoading && (
+          <span>
+            <Spinner size="sm" />
+          </span>
+        )}
```

Comment:
Could just be me, but the spinner position seems slightly below the center.

Reply:
Also, what do you think about using $oncokb-blue for spinner color (same as LoadingIndicator).

Reply:
![image](https://github.com/oncokb/oncokb-transcript/assets/59149377/59c27d57-7252-4652-8503-a6105167525a)